tests/ttnn/unit_tests/operations/pool/test_maxpool2d.py

```diff
@@ -282,6 +283,10 @@ def test_max_pool2d_localrun(device, dtype, in_place, input_spec):
     ) = input_spec
     if (kernel_height > 5 or kernel_width > 5) and in_place and dtype == ttnn.bfloat8_b:
         pytest.skip("this case runs out of memory due to combination of large remote temp CB and large untilize out CB")
+    if input_spec[:4] == [1, 512, 10, 10] and in_place and dtype == ttnn.bfloat8_b and is_blackhole():
+        pytest.skip(
+            "this case runs out of memory on blackhole due to combination of large remote temp CB and large untilize out CB"
```

Comment:
let's add issue in reference to this skip, and lets investigate this further we shouldn't have OOM cases on BH that run on WH